tools/ci/deps-ubuntu.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+# Code between the following doxygen markers are included in the
+# public-facing OpenMS installation instructions.
+
+# [installation_documentation]
+# Add "universe" and update:
+sudo add-apt-repository universe
+sudo apt update
+
+# Required dependencies:
+sudo apt-get -qq install -y \
+  build-essential \
+  cmake \
+  autoconf \
+  patch \
+  libtool \
+  git \
+  automake \
+  ninja-build \
+  xvfb \
```

Comment:
This is only needed if the computer does not have a monitor connected.

Reply:
Apparently this is a common enough use case that we need to support it.  We discussed this on Discord and people still build OpenMS on clusters and then remotely run the graphical applications via X11 forwarding via SSH.